config/authorization/config.ex

```diff
@@ -123,6 +94,31 @@ defmodule Acl.UserGroups.Config do
       } "
   end
 
+  defp is_admin() do
+    %AccessByQuery{
+      vars: [],
+      query: "PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+
+        SELECT DISTINCT ?session_role WHERE {
+          VALUES ?session_role {
+            \"LoketLB-admin\"
+          }
+
+          VALUES ?session_id {
+            <SESSION_ID>
+          }
+
+          {
+            ?session_id ext:sessionRole ?session_role .
+          } UNION {
+            ?session_id ext:originalSessionRole ?session_role .
+          }
+          BIND(COALESCE(?maybe_original_session_role, ?current_session_role) AS ?session_role)
+        }
+        LIMIT 1"
```

Comment:
I think we can simply remove the `BIND(COALESCE( ...` since it references things that don't exist anymore after I did some renaming. 

I don't understand why the query doesn't error though, since the spec says that it should if both variables are unbound: https://www.w3.org/TR/sparql11-query/#func-coalesce

> If none of the arguments evaluates to an RDF term, an error is raised. If no expressions are evaluated without error, an error is raised.

Reply:
I wanted to check this out of curiosity, and it seems it errors out when running it directly in the virtuoso sparql interface. This line has been removed now, but it's good to note in the future (maybe `mu-auth` does some re-writing before executing the query which caused the difference in behavior?).